pom.xml

```diff
@@ -14,6 +14,11 @@
   <name>Jenkins SSH Slaves plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Slaves+plugin</url>
 
+  <properties>
+    <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
+    <findbugs-maven-plugin.failOnError>true</findbugs-maven-plugin.failOnError>
```

Comment:
There is nothing to think about IMHO. the existing property used to control this configuration option is called `findbugs.failOnError` as per the plugins own [documentation](http://gleclaire.github.io/findbugs-maven-plugin/check-mojo.html#failOnError). 

I might have personal complaints about that specific plugin having picked a crummy name for the property... but that does not change the fact that the convention of all maven users is to use the property described in the plugin's documentation.

So the name should be `findbugs.failOnError` (even though I do not like that specific property name on so many levels... those complaints are to be addressed with the findbugs plugin maintainers)